src/com/shop/Shop.java

```diff
@@ -1,324 +1,335 @@
+			queryResult.next();
+			System.out.printf("Budget is: %.2f%n", queryResult.getDouble("budget"));
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private void createCustomer() {
+		System.out.println("Enter Name: ");
+		String customerName = input.next();
+		System.out.print("Enter Budget: ");
+		double customerBudget = input.nextDouble();
+		System.out.println();
+
+		try {
+			queryResult = selectAllCustomers.executeQuery();
+			do {
+				if (!queryResult.next()) {
+					break;
+				}
+				id = queryResult.getInt("cid") + 1;
+			} while (true);
+
+			insertCustomerRow.setInt(1, id);
+			insertCustomerRow.setString(2, customerName);
+			insertCustomerRow.setDouble(3, customerBudget);
+			queryResult = insertCustomerRow.executeQuery();
+
+			System.out.println("===== New Customer Was Created. =====");
+			System.out.println("Customer ID: " + id + " | Name: " + customerName + " | Budget: " + customerBudget + "\n");
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	public static void main(String[] args) {
+		Shop shop = new Shop();
+		String menuOption = "";
+		Connection conn = shop.connect();
+		
+		if (conn != null) {
+			shop.setPreparedStatements(conn);
+			do {
+				System.out.print("Enter Customer ID: ");
+				shop.id = shop.input.nextInt();
+				
+				if (shop.id == -1) {
+					shop.createCustomer();
+					break;
+				}
+			} while (!shop.isCustomerVerified());
+
+			while (!menuOption.equalsIgnoreCase("x")) {
+				shop.displayMenu();
+				menuOption = Character.toString(shop.input.next().charAt(0));
+				System.out.println();
+				
+				switch(menuOption) {
+					case "p", "P": {
+						shop.menuP();
+						break;
+					}
+					case "o", "O": {
+						shop.menuO();
+						break;
+					}
+					case "r", "R": {
+						shop.menuR();
+						break;
+					}
+					case "s", "S": {
+						shop.menuS();
+						break;
+					}
+					case "e", "E": {
+						shop.menuE();
+						break;
+					}
+					case "b", "B": {
+						shop.menuB();
+						break;
+					}
+					default: {
+						System.out.println("Invalid option");
+					}
+				}
+			}
+
+			try {
+				System.out.println("Connection closed.");
+				if (shop.queryResult != null) {
+					shop.queryResult.close();
+				}
+				if (shop.selectCustomerRow != null) {
+					shop.selectCustomerRow.close();
+				}
+				if (shop.selectSalesRow != null) {
+					shop.selectSalesRow.close();
+				}
+				if (shop.selectProductsRow != null) {
+					shop.selectProductsRow.close();
+				}
+				if (shop.updateBudget != null) {
+					shop.updateBudget.close();
+				}
+				if (shop.updateQuantity != null) {
+					shop.updateQuantity.close();
+				}
+				if (shop.selectProductsName != null) {
+					shop.selectProductsName.close();
+				}
+				if (shop.selectTotalCost != null) {
+					shop.selectTotalCost.close();
+				}
+				conn.close();
+			} catch (SQLException e) {
+				e.printStackTrace();
+			}
+
+		} else {
+			System.out.println("Connection FAILED.");
+		}
+	}
```

Comment:
Used Eclipse shortcut control+shift+f to auto format entire class to remove extra whitespace